fix/rJedit_Add_MutaFlag_doc.py

```diff
@@ -0,0 +1,79 @@
+def Add_MutaFlag(self,Flag = 'r', if_U = 0, if_self=0):
+        """Returns a label of mutations.
+
+        Reads input of single string or list of strings. Generates a random 
+        flag if input is "random" or "r". Appends self.MutaFlags with given
+        flag.
+
+        Args:
+            Flag: A string or a list of strings. 'r' or 'random' chooses a
+              random target residue mutation.
+
+                Example input:
+
+                X: original residue letter. X denotes unknown residue.
+                A: chain index.
+                11: residue index.
+                Y: target residue mutation.
+
+            if_U: include mutations to U (selenocysteine) in random generation 
+            if_self: for random generation, compares mutation to original 
+              residue.
+
+        Returns:
+            A list of strings that represent the original residue letter,
```

Comment:
talk about real return. And also the self.MutaFlags contains a list of tuples that contains strings inside of them. It's also useful to give examples.

Reply:
The idea is to make sure others know how to use these outputs without reading the code.